modules/c++/gsl/include/gsl/Gsl_.h

```diff
@@ -66,12 +66,20 @@ namespace Gsl
         {
             return (static_cast<U>(t) != u) ? narrow_throw_exception(t) : t;
         }
+
+        #if defined(_MSC_VER) && _MSC_VER <= 1900
```

Comment:
@kdwilhelm, `modules/c++/config/include/config/compiler_extensions.h` now has some cross-platform macros for doing this sort of thing  look for `CODA_OSS_disable_warning_push` in the file